pootle/checks.py

```diff
@@ -415,3 +415,19 @@ def check_email_server_is_alive(app_configs=None, **kwargs):
         else:
             connection.close()
     return errors
+
+
+@checks.register()
+def check_revision(app_configs=None, **kwargs):
+    from pootle.core.models import Revision
+    from pootle_store.models import Unit
+
+    errors = []
+    if Revision.get() is None or Revision.get() < Unit.max_revision():
+        errors.append(checks.Critical(
+            _("Revision is missed or has incorrect value."),
+            hint=_("Run `revision` command with `--restore` option."),
```

Comment:
Run `revision --restore` to reset the revision counter. (Or something like that, this way you can just copy the command instead of stitching it together).